src/app/confirmationDialog/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+    <div mat-dialog-content class="confirmationMessage">
+        Do you confirm the deletion of this Action Item?
```

Comment:
**SIDENOTE:** Normally all the texts in the applications are not putting text in the templates, but using the translation mechanism, you can check eg. here https://www.codeandweb.com/babeledit/tutorials/how-to-translate-your-angular8-app-with-ngx-translate (you don't need to do that in this PR).